website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -48,6 +48,7 @@ v0.34.0+ _only_ supports Karpenter v1beta1 APIs and will not work with existing
   * `Multi-Node Consolidation`: max 100 nodes
 * Karpenter now adds a default `podSecurityContext` that configures the `fsgroup: 65536` of volumes in the pod. If you are using sidecar containers, you should review if this configuration is compatible for them. You can disable this default `podSecurityContext` through helm by performing `--set podSecurityContext=null` when installing/upgrading the chart.
 * The `dnsPolicy` for the Karpenter controller pod has been changed back to the Kubernetes cluster default of `ClusterFirst`. Setting our `dnsPolicy` to `Default` (confusingly, this is not the Kubernetes cluster default) caused more confusion for any users running IPv6 clusters with dual-stack nodes or anyone running Karpenter with dependencies on cluster services (like clusters running service meshes). If you still want the old behavior here, you can change the `dnsPolicy` to point to use `Default` by setting the helm value on install/upgrade with `--set dnsPolicy=Default`. More details on this issue can be found in the following Github issues: [#2186](https://github.com/aws/karpenter-provider-aws/issues/2186) and [#4947](https://github.com/aws/karpenter-provider-aws/issues/4947).
+* Karpenter now isallows `nodepool.spec.template.spec.resources` to be set.
```

Comment:
Can you be more specific that this was never allowed with webhooks and that we are making a fix to ensure that this is the case with CEL as well. You should also, ideally, have a call to action here. Something like, "if you were previously setting the resources field on your NodePool, ensure that you remove this field before upgrading to the newest version of Karpenter or else updates to the resource may fail on the new version."